src/claimtrie.cpp

```diff
@@ -238,11 +198,9 @@ void CClaimTrie::clear(CClaimTrieNode* current)
 bool CClaimTrie::haveClaim(const std::string& name, const COutPoint& outPoint) const
 {
     const CClaimTrieNode* current = &root;
-    for (std::string::const_iterator itname = name.begin(); itname != name.end(); ++itname)
-    {
+    for (std::string::const_iterator itname = name.begin(); itname != name.end(); ++itname) {
```

Comment:
If this style change (and others like it) isn't pulled in from clang-format, remove it.

Reply:
Update: Hopefully this has already been clang-formatting because a large portion of it is just mechanical style changes, and it's distracting for such a large commit.  I realize that as we're requiring clang-format now this should be less an issue in the future.  If these were added manually, I suggest an immediate format/rebase.

Reply:
This is produced from clang-format?